### PR TITLE
Add integration tests kopia wrappers

### PR DESCRIPTION
**Pull request by ankitjain235**
## Change Overview

This PR adds integration tests for kopia utilities

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #issue-number

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/testing/kopia_cmd_test.go [hunk 1]
```diff
@@ -0,0 +1,229 @@
+//go:build kopia
+// +build kopia
+
+// Copyright 2022 The Kanister Authors.
```
Comment (PrasadG193) [inline pkg/testing/kopia_cmd_test.go:4]:
```suggestion
// Copyright 2023 The Kanister Authors.
```

**Review by PrasadG193 — COMMENTED** [review]
LGTM

**Comment by pavannd1** [comment]
@ankitjain235 Let's reopen this if relevant

**Comment by pavannd1** [comment]
@ankitjain235 Let me know if this is still required. We can close it if not

**Comment by ankitjain235** [comment]
Not complete as of now. We still need it but with including other wrapper tests as well. Closing it for now, we can re-open it later.

**Pull request closed without merging**